### DISCUSSION TITLE
Slider's behavior on Firefox - bug

### DISCUSSION THREAD

**Discussion by ClaraLouiseLBO** [armandsalle/butter-slider#39] [Q&A] [answered]
Hi Armand, thank you for sharing your Butter Slider! I've discovered your work through Alex Tourgis YT channel. 
Your slider is really cool and easy to install for someone like me, a noob in coding, so thank you for that!

I'm using the slider for a client's website, and I've found that it works nicely in Chrome, as exptected, but it seems that the drag & drop isn't working well with Firefox. Instead of sliding smoothly, it first doesn't slide when you click and drag the image, it tries to copy the image, and then the picture you clicked on kinda sticks to the cursor, and then only it starts sliding... 
I hope I made this clear.
If not, here is a link to the page I'm talking about, the slider is at the bottom of it : [here](https://corine-dol-martin.webflow.io/votre-psychologue).

Do you think you could help me figure out what's going on?

Thanks!

**Comment by armandsalle** [comment, marked as answer]
Hello 👋 Thank you for using BS! 

This is not a problem with BS, you need to prevent drag on image. You can do it with custom CSS, but with JS it's more efficient.
With this code, it should be working fine.

`document.querySelectorAll('img').forEach(img => img.setAttribute('draggable', false))`

Have nice day 🙌

**Reply by ClaraLouiseLBO** [reply]
Thanks for your answer, it works perfectly!
